### PR TITLE
feat: glide provider

### LINKED ISSUE #326: Glide image provider
I am working with a Laravel project using [Glide](https://glide.thephpleague.com/) image optimization to generate urls. I would like to contribute a Glide provider in the next couple days.

### PR DESCRIPTION
**Pull request by shadow81627**
https://glide.thephpleague.com/2.0/api/quick-reference/

resolves #326

### CODE REVIEW

File: src/runtime/providers/glide.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  joinWith: '&',
+  formatter: (key, val) => encodeQueryItem(key, val)
+})
+
+export const getImage: ProviderGetImage = (src, { modifiers = {}, baseURL = '/' } = {}) => {
+  const params = operationsGenerator(modifiers)
+
+  if (hasProtocol(src)) {
+    return {
+      url: src
```
Comment (pi0) [inline src/runtime/providers/glide.ts:25]: Why we need this condition?

**Comment by pi0** [comment]
Thanks 🙌 Would you mind also updating playground and docs to add new provider?

File: src/runtime/providers/glide.ts (hunk 1 above)
Comment (shadow81627) [reply src/runtime/providers/glide.ts]: Sorry I copied the ipx provider and forgot to remove.

**Comment by shadow81627** [comment]
@pi0 

The only glide instance that I have hosted is the one I use at work... not sure if they would be super happy having it in examples. Do you have any suggestions for the best way to provide a glide instance? Maybe I can host one on Heroku?

**Comment by pi0** [comment]
We usually use examples in provider documentation. For glide i could find this:

https://glide.herokuapp.com/1.0/kayaks.jpg?w=1000&gam=.9&sharp=8

**Review by pi0 — APPROVED** [review]
Neat 👍

File: test/unit/providers.test.ts [hunk 2]
```diff
@@ -87,6 +88,17 @@ describe('Providers', () => {
     }
   })
 
+  test('glide', () => {
```
Comment (pi0) [inline test/unit/providers.test.ts:91]: /refactor note: We may auto generate tests with a loop (/cc @danielroe)

**Pull request merged**